src/Symfony/Component/Form/ChainRequestHandler.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Symfony\Component\Form;
+
+use Symfony\Component\Form\Exception\LogicException;
+
+/**
+ * A request handler that chains over multiple request handlers,
+ * until it finds one that can handle the request.
+ *
+ * @author Wouter J <wouter@wouterj.nl>
+ */
+class ChainRequestHandler implements RequestHandlerInterface
```

Comment:
Could you rename this to RequestHandlerChain? This would be consistent with other such classes in the component (DataTransformerChain, FormTypeGuesserChain). I know that we're inconsistent with the naming across the framework, but fixing that is a separate issue.